src/emc/usr_intf/gmoccapy/gmoccapy.glade

```diff
@@ -6190,7 +6190,7 @@ F12 or $ key does the same</property>
               <object class="GtkLabel" id="lbl_ntb_button_manuel">
                 <property name="visible">True</property>
                 <property name="can-focus">False</property>
-                <property name="label">manuel</property>
+                <property name="label">manual</property>
```

Comment:
please review

Reply:
I would suspect lbl_ntb_button_manuel need to change too?  I can not find any traces of any use of the property or the label, so I guess it is safe to rename only here.

Reply:
I'm having issues pushing to my branch. @petterreinholdtsen feel free to push the changes and merge if you deem it worthy ?